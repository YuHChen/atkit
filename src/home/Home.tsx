import React from "react";

import Wip from "../Wip";

const Home = () => (
  <div>
    <div className="p-5 mb-4 bg-body-tertiary rounded-3">
      <h1 className="display-5">Welcome to atkit!</h1>
      <p>
        atkit is a toolkit for{" "}
        <a
          href="http://www.animationthrowdowngame.com"
          className="text-decoration-none"
        >
          Animation Throwdown: The Quest for Cards
        </a>{" "}
        game.
      </p>
    </div>

    <Wip pageName="website" />
  </div>
);

export default Home;
