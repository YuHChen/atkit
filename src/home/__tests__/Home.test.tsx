import React from "react";
import { render } from "@testing-library/react";

import Home from "../Home";

describe("Home", () => {
  test("renders as expected", () => {
    const { container } = render(<Home />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
