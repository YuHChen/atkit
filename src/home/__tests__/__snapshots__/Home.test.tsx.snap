// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Home > renders as expected 1`] = `
<div>
  <div
    class="p-5 mb-4 bg-body-tertiary rounded-3"
  >
    <h1
      class="display-5"
    >
      Welcome to atkit!
    </h1>
    <p>
      atkit is a toolkit for
       
      <a
        class="text-decoration-none"
        href="http://www.animationthrowdowngame.com"
      >
        Animation Throwdown: The Quest for Cards
      </a>
       
      game.
    </p>
  </div>
  <div
    class="fade alert alert-info show"
    role="alert"
  >
    <div
      class="alert-heading h4"
    >
      Pardon the mess
    </div>
    <p
      class="mb-0"
    >
      This 
      website
       is still a work in progress.
    </p>
  </div>
</div>
`;
